### PR TITLE
chore: create field-highlighter wrapper to avoid CE release

### PR DESCRIPTION
**Pull request by ZheSun88**
## Description

add a wrapper for filed-highlighter and let CE depends on this module.
so that we dont need to update CE with every WC version bump

### CODE REVIEW

File: scripts/mergeITs.js [hunk 1]
```diff
@@ -31,7 +31,9 @@ async function computeModules() {
   } else {
     // Read modules from the parent pom.xml
     const parentJs = await xml2js.parseStringPromise(fs.readFileSync(`pom.xml`, 'utf8'));
-    modules = parentJs.project.modules[0].module.filter(m => !/shared-parent/.test(m)).filter(m => !/demo-helpers/.test(m));
+    modules = parentJs.project.modules[0].module.filter(m => !/shared-parent/.test(m))
```
Comment (manolo) [inline scripts/mergeITs.js:34]: Probably we might invest some more time on adding a IT testing that the component works and it is initialised?

File: vaadin-field-highlighter-flow-parent/vaadin-field-highlighter-flow/src/main/java/com/vaadin/flow/component/fieldhighlighter/FieldHighlighter.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import com.vaadin.flow.component.dependency.JsModule;
+import com.vaadin.flow.component.dependency.NpmPackage;
+import com.vaadin.flow.dom.Element;
+import com.vaadin.flow.server.Command;
+import com.vaadin.flow.shared.Registration;
+
+@NpmPackage(value = "@vaadin/form-layout", version = "23.2.0-beta1")
+@JsModule("@vaadin/field-highlighter/src/vaadin-field-highlighter.js")
+public class FieldHighlighter {
```
Comment (web-padawan) [inline vaadin-field-highlighter-flow-parent/vaadin-field-highlighter-flow/src/main/java/com/vaadin/flow/component/fieldhighlighter/FieldHighlighter.java:26]: Let's change to `FieldHighlighterInitializer` as discussed internally.

File: vaadin-field-highlighter-flow-parent/vaadin-field-highlighter-flow/pom.xml [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            <plugin>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-maven-plugin</artifactId>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-jar-plugin</artifactId>
+                <configuration>
+                    <archive>
+                        <manifestFile>&#xD;
```
Comment (web-padawan) [inline vaadin-field-highlighter-flow-parent/vaadin-field-highlighter-flow/pom.xml]:
```suggestion
                        <manifestFile>
```

File: scripts/mergeITs.js [hunk 4]
```diff
@@ -31,7 +31,8 @@ async function computeModules() {
...
+      .filter(m => !/demo-helpers/.test(m));
```
Comment (web-padawan) [inline scripts/mergeITs.js:35]:
```suggestion
    modules = parentJs.project.modules[0].module.filter(m => !/shared-parent/.test(m)).filter(m => !/demo-helpers/.test(m));
```

**Review by web-padawan — APPROVED** [review]

**Pull request merged**